structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,201 @@
+
+Several relevant votes were held on the Program's Discord server leading up to the production of this document. They are as follows:
+
+1. When voting on whether to add or remove a community, how should the votes be counted?
+    - **10 answered:** One vote per community; each community will be required to formalize an internal voting process to figure out what their vote should be, and it must be democratic and fair
+    - **4 answered:** One vote per collaborator; each user or plural system present on the Discord server gets one vote
+
+2. What ratio of votes should be required to pass a membership vote?
+    - **11 answered:** > 75% approval to pass
+    - **4 answered:** > 50% approval to pass
+    - **1 answered:** 100% approval to pass
+
+3. For membership votes, should we have veto rights? If so, how should that work?
+   - **17 answered:** Any voter can veto, which can be overruled by the majority if the concerns aren't considered worthwhile enough to fail the vote
+   - **0 answered:** Any voter can veto, which will immediately fail the vote
+   - **0 answered:** There should be no veto rights
+
+ **TODO: THIS ISN'T DONE YET, RETURN ON THE 27TH OCT 2022**
+4. Should we allow for external collaborators that don't represent any specific community?
+    - **20 voted:** Yes
+    - **4 voted:** I don't care
+    - **0 voted:** Not
+
+# Prior art
+
+The Quilt Community Collab Program is a spiritual successor to an older program named Discord INTERPOL, a now-defunct private Discord server. This program intended to provide a way for notable Discord communities in the Fabric sphere to share information on incoming community threats and inform its members on issues of moderation. 
+
+Discord INTERPOL had many similarities with the Quilt Community Collab Program, including many of its foci and members. However, it lacked the support it needed from larger communities and didn't have the backing of a mod-loader project, which made it difficult to grow to a size that would see it become more widely useful.
+
+Additionally, the Quilt Community Collab Program has a much broader focus --- attempting to educate its staff on running their communities, and providing spaces to talk about general community management, defensive moderation tooling, wider social issues, and event planning.
+
+# Unresolved questions
+
+For the sake of getting this process document completed within a reasonable amount of time, the following concerns will need to be addressed at a later date:
+
+* We currently haven't discussed the possibility of opening Program membership to non-moderational community staff
+* We haven't discussed how new Managers should be added, which is an issue due to the fact that all Program Managers are also Quilt Community Managers; a more representative level of power-sharing should be worked on
```

Comment:
Assuming that some process exists whereby collaborators of the Program can become Program Managers, if a new Quilt Community Manager is elected by the process of the Quilt team, do they automatically assume a position as a Program Manager?

Reply:
This is currently undefined, let's not spend time on this right now.